hibernate6-jpa/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'io.micronaut.build.internal.sql-module'
+}
+
+// TODO: Temporary disabled tests and binary compatibility check
+test.onlyIf { false }
+micronautBuild {
+    binaryCompatibility {
+        enabled = false
+    }
+}
+
+dependencies {
+    annotationProcessor(mn.micronaut.graal)
+
+    // included for Java 11
+    runtimeOnly(libs.jaxb.api)
+
+    api(libs.managed.hibernate6.core) {
+        exclude group:'org.jboss.spec.javax.transaction', module:'jboss-transaction-api_1.2_spec'
```

Comment:
This `javax` exclude might be not needed

Reply:
Removed